packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -1206,4 +1257,121 @@ test.describe('PieInput - Component tests', () => {
+                    // Arrange
+                    const component = await mount(PieInput, {
+                        props: {} as InputProps,
+                    });
+
+                    // Act
+                    const input = component.locator('input');
+
+                    const componentAttribute = await input.getAttribute('aria-describedby');
+
+                    // Assert
+                    expect(componentAttribute).toBeNull();
+                });
+            });
+            test.describe('when `assistiveText` is defined', () => {
+                test('should render the attribute correctly with the correct value', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieInput, {
+                        props: {
+                            assistiveText: 'Some useful message',
+                        } as InputProps,
+                    });
+
+                    // Act
+                    const input = component.locator('input');
+
+                    const componentAttribute = await input.getAttribute('aria-describedby');
+
+                    // Assert
+                    expect(componentAttribute).toBeDefined();
+                });
+            });
+        });
+
+        test.describe('aria-invalid', () => {
+            test.describe('when the component status is set to `Error`', () => {
+                test('should render the `aria-invalid` attribute', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieInput, {
+                        props: {
+                            status: 'error',
+                        } as InputProps,
+                    });
+
+                    // Act
+                    const input = component.locator('input');
+
+                    const componentAttribute = await input.getAttribute('aria-invalid');
+
+                    // Assert
+                    expect(componentAttribute).toBeDefined();
+                });
+            });
+            test.describe('when the component `status` is set to anything but `Error`', () => {
+                test('should not render the should render the `aria-invalid` attribute', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieInput, {
+                        props: {
+                            status: 'success',
+                        } as InputProps,
+                    });
+
+                    // Act
+                    const input = component.locator('input');
+
+                    const componentAttribute = await input.getAttribute('aria-describedby');
+
+                    // Assert
+                    expect(componentAttribute).toBeNull();
+                });
+            });
+        });
+
+        test.describe('aria-errormessage', () => {
+            test.describe('when the component status is set to `Error`', () => {
```

Comment:
`error` might be a little more accurate - same for all other references in this file